sdk/communication/communication-chat/test/chatThreadClient.spec.ts

```diff
@@ -113,7 +113,21 @@ describe("ChatThreadClient", function() {
   it("successfully lists participants", async function() {
     const list: string[] = [];
```

Comment:
shouldn't we assert something about this list?

Reply:
We should. Leaving that to @LuChen-Microsoft and friends